.circleci/config.yml

```diff
@@ -2,15 +2,12 @@
 #
 # Check https://circleci.com/docs/2.0/language-python/ for more details
 #
-version: 2
+version: 3
```

Comment:
I don't think there is a version 3 for CircleCI configs: https://circleci.com/docs/2.0/configuration-reference/